raiden/raiden_event_handler.py

```diff
@@ -608,6 +676,37 @@ def handle_contract_send_channelunlock(
                     partner=our_address,
                 )
 
+    @staticmethod
+    def handle_contract_send_channelunlock_light(
+        raiden: "RaidenService",
+        chain_state: ChainState,
+        channel_unlock_event: ContractSendChannelBatchUnlockLight,
+    ):
+        channel_state = get_channel_state(
+            raiden=raiden,
+            chain_state=chain_state,
+            canonical_identifier=channel_unlock_event.canonical_identifier,
+            participant=channel_unlock_event.participant,
+            our_address=channel_unlock_event.client)
+        if should_search_events(channel_state):
```

Comment:
I think you could put these 2 if statements as just one, since those are both identical. Maybe something like this:

```python
if should_search_events(channel_state) or should_search_state_changes(channel_state):
            unlock_light(
                raiden=raiden,
                chain_state=chain_state,
                channel_unlock_event=channel_unlock_event,
                participant=channel_state.partner_state.address,
                partner=channel_state.our_state.address,
                end_state=channel_state.our_state
            )
```

Reply:
They are (sadly :( )not identical, the partner and the participant are swapped between each other

Reply:
Oh ok sorry didn't saw it the first time.